packages/blocks/src/store/selectors.js

```diff
@@ -96,3 +132,22 @@ export const getChildBlockNames = createSelector(
 export const hasChildBlocks = ( state, blockName ) => {
 	return getChildBlockNames( state, blockName ).length > 0;
 };
+
+/**
+ * Returns a boolean indicating if a block has at least one child block with inserter support.
+ *
+ * @param {Object} state     Data state.
+ * @param {string} blockName Block type name.
+ *
+ * @return {boolean} True if a block contains at least one child blocks with inserter support and false otherwise.
+ */
+export const hasChildBlocksWithInserterSupport = createSelector(
+	( state, blockName ) => {
+		return some( getChildBlockNames( state, blockName ), ( currentBlockName ) => {
+			return hasBlockSupport( state, currentBlockName, 'inserter', true );
```

Comment:
Curious: Do we use this `true` value for `'inserter'` supports elsewhere? Wondering at what point we'll start running into issues of magic constants being used in multiple places.

Reply:
It was already being used in getInserterItems selector but it was not used in any other place.